apps/web/src/hooks/useFindSessionQuery.ts

```
@@ -0,0 +1,52 @@
+    params: {}
+  }
+) => {
+  return useQuery({
+    enabled,
+    queryFn: async () => {
+      const response = await axios.get('/v1/sessions', {
+        params
+      });
+      const parsedData = $SessionWithUser.array().safeParseAsync(response.data);
+      if ((await parsedData).error) {
+        console.log((await parsedData).error);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)